mtp_api/apps/performance/tests/test_views.py

```diff
@@ -159,8 +159,8 @@ def test_filtering_invalid_input(self):
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         body = response.json()
-        self.assertIn('Enter a valid date', body['week__gte'])
-        self.assertIn('Enter a valid date', body['week__lt'])
+        self.assertIn('Enter a valid date.', body['week__gte'])
+        self.assertIn('Enter a valid date.', body['week__lt'])
```

Comment:
same issue as in noms-ops
https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/617#discussion_r2007484383

Reply:
FYI: I've bumped mtp-common + [reverted that commit](https://github.com/ministryofjustice/money-to-prisoners-api/pull/821/commits/00df52ed23e9365943009c2abaaea8c9fb1e4549), so this should be sorted as soon as we manage to release mtp-common 19.0.1.